maapcopcguide (2).ipynb

```diff
@@ -0,0 +1,290 @@
+{
```

Comment:
<p>&gt;<strong>Authors</strong>: Harshini Girish (UAH), Rajat Shinde (UAH), Alex Mandel (DevSeed), Jamison French (DevSeed), Brian Freitag (NASA MSFC), Sheyenne Kirkland (UAH)</p><p> </p><p>We can include Chuck, Henry, Zac as co-authors</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/459/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>